eng/service.proj

```diff
@@ -51,6 +51,14 @@
              SkipNonexistentTargets="true" />
   </Target>
 
+  <Target Name="GenerateTest">
```

Comment:
Nit: "GenerateTests" (plural) sounds more apt. Your call to `Write-Host` above even uses the plural form since we'll certainly generate numerous tests.

Reply:
done